views/page/index.html

```diff
@@ -10,52 +10,57 @@
 {% set banner = true %}
 
 {% block content %}
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-            <div class="header-search">
-                <h1 class="govuk-heading-l">Content management</h1>
-                {{ search() }}
-            </div>
+{% set errors = sessionFlash.errors %}
+
+{{ errorSummary(errors.fields) }}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <div class="header-search">
+            <h1 class="govuk-heading-l">Content management</h1>
+            {{ search() }}
         </div>
     </div>
+</div>
 
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
         {% if pageResults %}
             {% if pageResults.totalResults !== 0 %}
                 {% block list %}
-                    {{ list("Courses", pageResults) }}
+                {{ list("Courses", pageResults) }}
                 {% endblock %}
-            {% else %}
+                {% else %}
```

Comment:
Looks like the indentation has changed here, should probably be as it was before